### PR TITLE
Feature/proof request

### PR DESCRIPTION
**Pull request by manghuli**
# Summary of Changes

Replace this text with a high-level summary of the changes included in this PR.

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [ ] Run prettier: `npm run style-format`
- [ ] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by TimoGlastra — DISMISSED** [review]
Nice! A lot to review in a single PR, but I like that we're updating to the latest AFJ version. 

I'd be interested to see a demo of this new functionality at the WG call!

File: App/screens/CredentialDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      title: route?.params?.alias,
+    })
+  }, [])
+
+  return (
+    <SafeAreaScrollView>
+      <Text>{first_name}</Text>
+      <Text>{last_name}</Text>
+      <Label title="Gender" subtitle={gender} />
+      <Label title="Age" subtitle={age} />
```
Comment (JamesKEbert) [inline App/screens/CredentialDetails.tsx]: This is something that we will need to fix in the future--this needs to be generic and not tied to a specific set of attributes, but I'd rather merge and do a followup PR.

File: App/screens/EntryPoint.tsx [hunk 2]
```diff
@@ -3,17 +3,17 @@ import { useHistory } from 'react-router-native'
 import * as Keychain from 'react-native-keychain'
 import AsyncStorage from '@react-native-community/async-storage'
 
-import LoadingOverlay from '../LoadingOverlay/index'
+import LoadingOverlay from '../components/modals/LoadingOverlay'
```
Comment (JamesKEbert) [inline App/screens/EntryPoint.tsx]: I think we _might_ not need this component?

File: App/screens/Home.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+} from 'components'
+
+interface Props {
+  navigation: any
+}
+
+const Home: React.FC<Props> = ({ navigation }) => {
+  const credentials = useCredentialByState(CredentialState.OfferReceived)
+  const proofs = useProofByState(ProofState.RequestReceived)
+  console.log('PROOFS', proofs)
```
Comment (JamesKEbert) [inline App/screens/Home.tsx]: Unneeded console log?

File: App/screens/ProofRequest.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+const transformAttributes = (attributes: any) => {
+  const transformedAttributes = []
+
+  for (const attribute in attributes) {
+    transformedAttributes.push({
+      name: attribute,
+      value: attributes[attribute][0].credentialInfo.attributes[attribute],
+      credentialDefinitionId: attributes[attribute][0].credentialInfo.credentialDefinitionId
```
Comment (JamesKEbert) [inline App/screens/ProofRequest.tsx]: Would prefer to take the schema name for this to mimic the credential offer/issuance.

Comment (JamesKEbert) [reply App/screens/ProofRequest.tsx]: And this can use the parseSchema function too

File: aries-hooks/package.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "aries-hooks",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.tsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (JamesKEbert) [inline aries-hooks/package.json]: We need to make sure this "Apache-2.0" for the license.

**Review by JamesKEbert — CHANGES_REQUESTED** [review]
Looks great! A couple of minor tweaks requested--and can be resolved with the merge conflicts.

File: App/screens/CredentialDetails.tsx (hunk 1 above)
Comment (manghuli) [reply App/screens/CredentialDetails.tsx]: ya, we still aren't using this component at all, it's just a placeholder for now

Comment (JamesKEbert) [reply App/screens/CredentialDetails.tsx]: Ahh I see 👍

File: App/screens/ProofRequest.tsx (hunk 4 above)
Comment (manghuli) [reply App/screens/ProofRequest.tsx]: i went through and couldn't find a schemaId on that object anywhere, but I'll check again

File: App/screens/EntryPoint.tsx (hunk 2 above)
Comment (manghuli) [reply App/screens/EntryPoint.tsx]: removed

File: App/screens/Home.tsx (hunk 3 above)
Comment (manghuli) [reply App/screens/Home.tsx]: removed

File: App/screens/ProofRequest.tsx (hunk 4 above)
Comment (manghuli) [reply App/screens/ProofRequest.tsx]: oof, it was right there the whole time haha my bad

**Comment by JamesKEbert** [comment]
Would you be able to resolve the merge conflicts here @NeilSMyers?

**Review by JamesKEbert — APPROVED** [review]
Looks great! Appreciate your work here!

**Pull request merged**